### ISSUE TITLE
Accessibility Violations -  StackBarChart, AreaChart and StackedAreaChart

### ISSUE THREAD

**Issue by MichaelMichaelMotorcycle** [carbon-design-system/carbon-charts#752]
#### I am submitting a...

- [ ] Feature request
- [x] Design defect
- [ ] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
"@carbon/charts": "^0.35.1"

### Issue description
 Running the  "IBM Equal Access Accessibility Checker" on a AreaChart, StackedAreaChart and StackedBarChart has accessibility violations

#### Steps to produce the issue

1. Took sandbox code for any of the three 
  area--area-time-series
  https://carbon-design-system.github.io/carbon-charts/react/?path=/story/area--area-time-series
  https://codesandbox.io/s/yhcf1?file=/src/index.js:0-2142
  
  area--stacked-area-time-series:
  https://carbon-design-system.github.io/carbon-charts/react/?path=/story/area--stacked-area-time-series  
  https://codesandbox.io/s/1xe7q?file=/src/index.js:0-2085
  
  bar-vertical--stacked-bar-time-series
  https://carbon-design-system.github.io/carbon-charts/react/?path=/story/bar-vertical--stacked-bar-time-series
  https://codesandbox.io/s/ngg92
   
   
   moved files as such    ./src/index.js   ./src/ibm-plex-font.css   ./public/index.html   ./package.json
2. yarn install && yarn Dev 
3.  Install and run  "IBM Equal Access Accessibility Checker"  on the page.  I used the chrome extension here.
https://chrome.google.com/webstore/detail/ibm-equal-access-accessib/lkcagbfjnkomcinoddgooolagloogehp
4. In the browser web debugger run the accessibility checker 

#### Current behavior
We are seeing these in our application and can reproduce using the story book example data and configurations.
 
1. On the graph tool tip when it is hidden: 
"Text contrast of 1.00 with its background is less than the WCAG AA minimum requirements for text of size 12px and weight of 400 "

2. On the SVGs in the graph
"The 'role' defined on the element is not valid per WAI-ARIA specification"

roles that are present 
role="graphics-symbol"  and  role="graphics-document document"

Item 2 seems similar to https://github.com/carbon-design-system/carbon-charts/issues/536
but it should be fixed in version 0.35.1 we are using

#### Expected behavior

No accessibility violations in base carbon chart features for the AreaChart .

**Comment by theiliad** [comment]
> raphics-symbol

1. Hidden elements don't cause visual accessibility issues. This is probably a bug in DAP. @sophiiae we can try adding a `aria-hidden: true` when tooltip isn't visible
2. Those roles are valid roles, so basically it's a false positive on DAP's side. We can add the `img` role though @sophiiae as fallback so DAP would stop throwing warnings
